Modules/@babylonjs/react-native/shared/XrAnchorHelper.h

```diff
@@ -183,3 +183,47 @@ namespace Babylon::Plugins::NativeXr
 }
 #endif
 #endif
+
+#if __has_include("IXrContextARKit.h")
+#include "IXrContextARKit.h"
+#if __has_include("jsi/jsi.h")
+namespace Babylon::Plugins::NativeXr
+{
+    bool TryGetNativeAnchor(facebook::jsi::Runtime& jsiRuntime, facebook::jsi::Value& jsAnchor, ARAnchor*& nativeAnchor)

```

Comment:
I think I prefer to use reference to pointer, but I feel like the pattern in OpenXR/Vulkan/D3D is usually to do pointer to pointer. Idk which is actually better, just wanted to _point_ it out

Reply:
Also, without looking too hard I think the first two references could be const all the way down

Reply:
I agree on switching things that aren't getting set to constants. I think that a reference to a pointer is more legible than a double pointer personally.

Reply:
I am going to leave things as they are for now. jsiRuntime.global() isn't marked as a const operation so we can get away with handing around a const jsiRuntime here.